docs/_includes/helpers/code-example.html

```diff
@@ -1,5 +1,5 @@
 {{ include.code }}
 
 ```html
-{{ include.code }}
+{{ include.code | replace_regex: '\s?test-[^\s"]+', '' }}
```

Comment:
This removes `.test-*` classes from code samples, as they’re used for integration tests but aren’t relevant for consumers of the documentation.

(Using `.test-*` classes makes selecting targets in tests much more durable).

Reply:
I think it’s about the same in terms of risk of collision — the `data-test="name"` attribute can also be styled (but why would you?). I think the same rationale could be used for applying styles to `.test-name` classes.

That being said, I’m not married to any particular decision — the goal here was to be able to clearly select elements for testing purposes with selectors that aren’t brittle (for example, using `.test-accordion` instead of `.usa-prose .usa-accordion` (there’s also an accordion in the menu bar).

Reply:
Switched to `data-test="name"` in 063a2d1 — I don’t have a strong preference, so I’ll go forward with your suggestion!